parsers.go

```diff
@@ -10,14 +18,8 @@ func parseIssueComment(body []byte) (IssueComment, error) {
 				URL string `json:"url"`
 			} `json:"pull_request"`
 		} `json:"issue"`
-		Repository struct {
-			Name  string `json:"name"`
-			Owner struct {
-				Login string `json:"login"`
-			} `json:"owner"`
-			SSHURL string `json:"ssh_url"`
-		} `json:"repository"`
-		Comment struct {
+		Repository messageRepository `json:"repository"`
+		Comment    struct {
```

Comment:
The nice thing about Go development is that we don't have to argue about any of this, because the formatting is [done automatically by gofmt](https://blog.golang.org/go-fmt-your-code).

But as for an explanation, you can see that gofmt aligns the types for all of the fields in a single struct. 4 spaces are required to have this `struct` type align with `messageRepository` just above it.